public/app/features/dashboard-scene/scene/NavToolbarActions.tsx

```diff
@@ -170,6 +172,49 @@ export function ToolbarActions({ dashboard }: Props) {
     ),
   });
 
+  toolbarActions.push({
+    group: 'playlist-actions',
+    condition: isPlaying && !editview && !isEditingPanel && !isEditing,
```

Comment:
Nit: Can we extract this condition to a const to not repeat it a few times down below?